compiler/src/iree/compiler/Pipelines/Options.cpp

```diff
@@ -49,8 +49,8 @@ void InputDialectOptions::bindOptions(OptionsBinder &binder) {
                      "Legalize from MHLO ops.")
         , clEnumValN(InputDialectOptions::Type::xla, "xla",
             "Legalize from MHLO ops (with XLA cleanup preprocessing).")
-        , clEnumValN(InputDialectOptions::Type::stablehlo_experimental,
-            "stablehlo_experimental",
+        , clEnumValN(InputDialectOptions::Type::stablehlo,
+            "stablehlo",
             "Legalize from StableHLO ops. WARNING: This is work in progress.")
```

Comment:
maybe drop the warning?

```suggestion
        , clEnumValN(InputDialectOptions::Type::stablehlo,
            "stablehlo", "Legalize from StableHLO ops.")
```

Reply:
I considered this and decided to keep it until we make it the default input conversion pipeline. This is because we do know that there are a few missing things.